viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -736,6 +736,13 @@ export class Cognite3DViewer {
       preMultipliedRotation
     );
 
+    const numberOf360Images = image360Collection.image360Entities.length;
+
+    MetricsLogger.trackEvent('360ImageCollectionAdded', {
+      datasource,
+      numberOf360Images
+    });
```

Comment:
If `numberOf360Images` is `0`, should we track the event as `360ImageCollectionAdded`

Reply:
sure, why not? Gives us a good metric on how often people send in and invalid filter, which is good to know.

Reply:
👍